test/harness/jpype/proxy/Wait.java

```diff
@@ -0,0 +1,16 @@
+package jpype.proxy;
+
+public class Wait {
+    public static void snooze() throws Throwable {
+	    try {
+	    System.out.println("Waiting");
+	    Thread.sleep(1000);
+	    System.out.println("Wake");
+	    }
+	    catch (Throwable th)
+	    {
+		    System.out.println("Catch "+th);
+		    throw th;
```

Comment:
is this exception handled elsewhere now? Why the rethrow?

Reply:
Wait is a test code that I was using to print a count down..  I will remove it.